### ISSUE TITLE
Retrieve data by using LIKE operator

### ISSUE THREAD

**Issue by gdonato022** [praeclarum/sqlite-net#1262]
I'm trying to execute this query, but although it's not throwing an exception, I'm not getting any data (I'm sure there's data in the database).

```
 var sql = new StringBuilder(
     $@"
       SELECT * 
       FROM Airport 
       WHERE 
         
         AirportCity LIKE '%' + ? + '%' "
 );

var parameters = new List<object> { searchTerm };

db => db.QueryAsync<Airport>(sql.ToString(), parameters.ToArray())
```

**Comment by gilzoide** [comment]
Hey @gdonato022. In SQLite, string concatenation uses the `||` operator, not `+`. Have you tried it?

**Comment by TheWatchfulOne** [comment]
I don't think you need to concatenate anything. SQLite will replace the question mark with the criteria value you supply it.  Try this:
`var sql = new StringBuilder(
     $@"
       SELECT * 
       FROM Airport 
       WHERE 
         
         AirportCity LIKE '%?%' "
 );`
My disclaimer is I'm not familiar the implications of using StringBuilder. I think you could just declare a string unless this what your application requires.
